src/VisualStudio/Core/Def/Implementation/ProjectSystem/IVisualStudioWorkspaceHost.cs

```diff
@@ -34,5 +34,6 @@ internal interface IVisualStudioWorkspaceHost
         void OnAdditionalDocumentOpened(DocumentId documentId, ITextBuffer textBuffer, bool isCurrentContext);
         void OnAdditionalDocumentClosed(DocumentId documentId, ITextBuffer textBuffer, TextLoader loader);
         void OnAdditionalDocumentTextUpdatedOnDisk(DocumentId id);
+        void OnIntellisenseBuildResult(ProjectId projectId, bool succeed);
```

Comment:
I'd also call it something more OnIsComplete changed or something, since I can imagine scenarios where we'd change it for reasons other than Intellisense builds working or not working.